### ISSUE TITLE
Feature request: don't hide toolbar buttons in drop-down menu unnecessarily

### ISSUE THREAD

**Issue by cfr42** [mozilla/pdf.js#9200]
Attach (recommended) or Link to PDF file here: N/A (feature request)

Configuration:
- Web browser and its version: FF 57.0 (64 did)
- Operating system and its version: Arch Linux: no version: updated from stable 2017-11-27
- PDF.js version: built into FF
- Is a browser extension: no - built-in

Steps to reproduce the problem: No problem. Just open a PDF.

What is the expected behaviour? (add screenshot) Behaves as expected. However, I would like the toolbar buttons to all be displayed on the toolbar, if space allows. That is, I would like some of them to drop off the bar into a context menu only if there is insufficient space for them.

What went wrong? (add screenshot) Nothing. But it seems odd to push buttons to a drop-down menu for no reason, when there is a huge amount of space available on the bar.

Link to a viewer (if hosted on a site other than mozilla.github.io/pdf.js or as Firefox/Chrome extension): N/A

**Comment by GauravV-02** [comment]
@cfr42 Hi, I am a student taking open source development course at Seneca college, looking for an issue to fix as a requirement of my course. Can I work on this?

Regards

**Comment by himanish-star** [comment]
@timvandermeij , I think I can solve this issue. If you permit, may I ?

**Comment by timvandermeij** [comment]
Since this is a change in the UI, I'd suggest talking this over with `yury` or `bdahl` on IRC first to check what should be done exactly and if this is something we want to fix,

**Comment by himanish-star** [comment]
@timvandermeij , yeah sure, I'll wait.

**Comment by himanish-star** [comment]
> I'd suggest talking this over with yury or bdahl on IRC first 

@timvandermeij , I might have got this wrong. Was I supposed to ask on this issue on the IRC or you ?

**Comment by timvandermeij** [comment]
You can ask on IRC if this is something that we want to fix.

**Comment by himanish-star** [comment]
@timvandermeij , no one is replying on the IRC, so can I look for other issues until I get some reply ?

**Comment by timvandermeij** [comment]
Yes, feel free to work on another issue until this one gets sorted out.
